scripts/zulip_emoji_merge_delegate.py

```diff
@@ -109,26 +129,29 @@ def remove_reaction(name: str, emoji_name: str, **kwargs) -> None:
             # 61282 was the earlier version of the emoji.
             remove_reaction('closed-pr', 'closed-pr', emoji_code="61293", reaction_type="realm_emoji")
 
-        # applying appropriate emoji reaction
+        # Apply the appropriate emoji reaction.
         print("Applying reactions, as appropriate.")
         def add_reaction(name: str, emoji_name: str) -> None:
-            print(f'adding {name}')
-            result = client.add_reaction({
+            print(f'adding {name} emoji')
+            client.add_reaction({
                 "message_id": message['id'],
                 "emoji_name": emoji_name
             })
-        if 'ready-to-merge' == LABEL:
-            add_reaction('ready-to-merge', 'bors')
-        elif 'delegated' == LABEL:
-            add_reaction('delegated', 'peace_sign')
-        elif 'maintainer-merge' == LABEL:
-            add_reaction('maintainer-merge', 'hammer')
-        elif LABEL == 'labeled':
-            add_reaction('awaiting-author', 'writing')
-        elif LABEL == 'closed':
-            add_reaction('closed-pr', 'closed-pr')
-        elif LABEL == 'unlabeled':
-            print('awaiting-author removed')
-            # The reaction was already removed.
-        elif LABEL.startswith("[Merged by Bors]"):
-            add_reaction('[Merged by Bors]', 'merge')
+        match ACTION:
+            case 'ready-to-merge':
+                add_reaction('ready-to-merge', 'bors')
+            case 'delegated':
+                add_reaction('delegated', 'peace_sign')
+            case 'LABELED':
```

Comment:
I'm guessing this should still be lower-case?
```suggestion
            case 'labeled':
```